File: VB/HowToImplementSingleton.Module.Web/WebModule.vb
Imports Microsoft.VisualBasic
Imports System
Imports System.ComponentModel

Imports DevExpress.ExpressApp

Namespace HowToImplementSingleton.Module.Web
	<ToolboxItemFilter("Xaf.Platform.Web")> _
	Public NotInheritable Partial Class HowToImplementSingletonAspNetModule
		Inherits ModuleBase
		Public Sub New()
			InitializeComponent()
		End Sub
	End Class
End Namespace

File: CS/HowToImplementSingleton.Module.Web/WebModule.cs
using System;
using System.ComponentModel;

using DevExpress.ExpressApp;

namespace HowToImplementSingleton.Module.Web {
	[ToolboxItemFilter("Xaf.Platform.Web")]
	public sealed partial class HowToImplementSingletonAspNetModule : ModuleBase {
		public HowToImplementSingletonAspNetModule() {
			InitializeComponent();
		}
	}
}
